external/fv3fit/tests/training/test_train.py

```diff
@@ -382,16 +389,18 @@ def test_dump_and_load_default_maintains_prediction(model_type):
     n_sample, n_tile, nx, ny, n_feature = 1, 6, 12, 12, 2
     sample_func = get_uniform_sample_func(size=(n_sample, n_tile, nx, ny, n_feature))
     result = train_identity_model(model_type, sample_func=sample_func)
-
     original_result = result.model.predict(result.test_dataset)
     with tempfile.TemporaryDirectory() as tmpdir:
         fv3fit.dump(result.model, tmpdir)
         loaded_model = fv3fit.load(tmpdir)
+    loaded_result = loaded_model.predict(result.test_dataset)
```

Comment:
For some reason, the loaded custom keras models can't call their `.summary()` until you use them to predict something, so I moved this line up to before the assertions.